internal/lint/lint_test.go

```diff
@@ -59,13 +67,23 @@ func TestLint(t *testing.T) {
 	}
 
 	t.Run("TestGolint", func(t *testing.T) {
+		// Go versions less than 1.17 do not support the `go run path@version`
```

Comment:
Alternatively - we could just drop CI support for go1.16, which would avoid the need for this version parsing.